applications/model-registry/upstream/base/model-registry-deployment.yaml

```diff
@@ -24,43 +24,12 @@ spec:
           args:
             - --hostname=0.0.0.0
             - --port=8080
-            - --mlmd-hostname=localhost
-            - --mlmd-port=9090
-            - --datastore-type=mlmd
+            - --datastore-type=embedmd
+            - --embedmd-database-dsn=$(DBCONFIG_USER):$(DBCONFIG_PASSWORD)@tcp($(MYSQL_HOST):$(MYSQL_PORT))/$(MYSQL_DATABASE)?charset=utf8mb4
           command:
             - /model-registry
             - proxy
           image: ghcr.io/kubeflow/model-registry/server:latest
-          # empty placeholder environment for patching
-          env: []
-          ports:
-            - name: http-api
-              containerPort: 8080
-          livenessProbe:
-            initialDelaySeconds: 30
-            periodSeconds: 5
-            tcpSocket:
-              port: http-api
-            timeoutSeconds: 2
-          readinessProbe:
-            initialDelaySeconds: 3
-            periodSeconds: 5
-            tcpSocket:
-              port: http-api
-            timeoutSeconds: 2
-          securityContext:
```

Comment:
Why are you removing the securitycontext? It must satisfy PSS restricted or the CI/Cd will fail.